src/BinSkim.Sdk/ICompilerDataLogger.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis.BinaryParsers.ProgramDatabase;
+
+namespace Microsoft.CodeAnalysis.IL.Sdk
+{
+    public interface ICompilerDataLogger
```

Comment:
>ICompilerDataLogger [](http://example.com/codeflow?start=21&length=19)

why did you create an interface?

Reply:
just deleted.
i thought it was going to be useful.